install/kubernetes/helm/subcharts/kiali/templates/configmap.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     release: {{ .Release.Name }}
 data:
   config.yaml: |
+    istio_namespace: {{ .Release.Namespace }}
```

Comment:
As Kiali was already installed in this namespace same as Istio, what is the use of the `istio_namespace` here?

Reply:
Kiali itself supports being installed in a different namespace than where Istio is located. This configuration setting is the way to tell Kiali where Istio is. Kiali in its implementation does not ask the cluster at runtime where it is installed (since it could be different than where Istio is) so Kiali relies on this config setting to tell it the namespace where Istio is located.